docker/cygnus-ngsi-ld/Dockerfile

```diff
@@ -20,7 +20,7 @@
 # as we did with cygnus-ngsi Dockerfile. Basically, the changes done in PR https://github.com/telefonicaid/fiware-cygnus/pull/2152
 # should be also applied here
```

Comment:
This FIXME block should be removed

Reply:
Done in https://github.com/telefonicaid/fiware-cygnus/pull/2153/commits/605de24548194859b097ab046b2ea126bed58591